jsonnet/components/openshift-state-metrics.libsonnet

```diff
@@ -18,6 +18,12 @@ function(params) {
   clusterRoleBinding: osm.openshiftStateMetrics.clusterRoleBinding,
   clusterRole: osm.openshiftStateMetrics.clusterRole,
   deployment: osm.openshiftStateMetrics.deployment {
+    metadata+: {
+      labels+: {
+        'app.kubernetes.io/managed-by': 'cluster-monitoring-operator',
+        'app.kubernetes.io/part-of': 'openshift-monitoring',
```

Comment:
This should be coming from `main.jsonnet` instead of being redefined here. It is defined in https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/main.jsonnet#L72-L74 and should be passed to openshift-state-metrics in the same way as https://github.com/openshift/cluster-monitoring-operator/blob/master/jsonnet/main.jsonnet#L103.